test/util/vectorization_test.py

```diff
@@ -0,0 +1,429 @@
+# Copyright 2014, 2015 The ODL development group
```

Comment:
No, because `3 * x` is again a `FunctionSpaceVector` which has something like `lincomb(3, old_vec)` as its `_call` implementation. Actually there are heaps of tests for this. Basically all the tests using `novec` functions first manually vectorize and then test arrays and meshgrid. Looked all good last time.